src/www/base.webpack.js

```diff
@@ -0,0 +1,78 @@
+// Copyright 2020 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const path = require('path');
+const CopyPlugin = require('copy-webpack-plugin');
+const WebpackShellPluginNext = require('webpack-shell-plugin-next');
+
+const OUTPUT_BASE = path.resolve(__dirname, '../../www');
+
+exports.makeConfig = (options) => {
+  return {
+    entry: [
+      require.resolve('@webcomponents/webcomponentsjs/webcomponents-bundle.js'),
```

Comment:
Let me move them to the CopyPlugin. I think that adding them here would be functionally equivalent to what we have because the loaders would still process them.

Reply:
Thanks for helping me figure this out.

To summarize:
* `custom-elements-es5-adapter.js` should not be transpiled (because it does an ES6 check) so we statically add it in `cordova_index.html` and provide it using the the `CopyPlugin`
* The rest of the polyfills are added to `cordova_main.ts` and `electron_main.ts`